tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -32,26 +32,16 @@ export function matMulReadFnSource(
       let batch = ${batchAEqualOne ? '0' : 'batchIn'};
       let batchASize = uniforms.aShape[1] * uniforms.aShape[2];
```

Comment:
nit: Remove `batchASize` since it's not needed anymore.

Reply:
Done.